doc/en/components/inputs/mask-input.md

```diff
@@ -0,0 +1,147 @@
+---
+title: $Platform$ MaskInput | Infragistics
+_description: Infragistics' $Platform$ MaskInput allows the user to control input and format the visible value based on configurable mask rules
+_keywords: $Platform$ input, $ProductName$, Infragistics
+mentionedTypes: ['MaskInput']
+---
+
+## $Platform$ Mask Input Overview
+
+The $ProductName$ Mask Input is an input field that allows the developer to control user input and format the visible value, based on configurable rules. It provides different input options and ease in use and configuration.
+
+### $Platform$ Mask Input Example
+
+<code-view style="height: 150px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/mask-input-overview"
+           alt="$Platform$ Mask Input Overview Example"
+           github-src="inputs/mask-input/overview">
+</code-view>
+
+## Usage
+
+<!-- WebComponents -->
+First, you need to install the $ProductName$ by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+<!-- end: WebComponents -->
+
+Before using the `IgcMaskInput`, you need to register it as follows:
+
+
+```ts
+import { defineComponents, IgcMaskInputComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcMaskInputComponent);
+```
+
+### Mask Rules
+The table bellow shows the supported built-in mask rules:
+
+| Mask Character | Description |
+| :--- | :--- |
+| 0 | requires a digit (0-9) |
+| 9 | requires a digit (0-9) or a space |
+| # | requires a digit (0-9), plus (+), or minus (-) sign |
+| L | requires a letter (a-Z) |
+| ? | requires a letter (a-Z) or a space |
+| A | requires an alphanumeric (0-9, a-Z) |
+| a | requires an alphanumeric (0-9, a-Z) or a space |
+| & | any keyboard character (excluding space) |
+| C | any keyboard character |
```

Comment:
I suggest using the table from the specification. Currently there is no distinction about which mask flags are required and which are optional.